tests/integration_tests/security/spectre-mitigations

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
nit: why bash and not python?

Reply:
It is a relatively short and simple script and simplifies many of the operations such as `grep -q "^(Mitigation: CSV2, BHB|Not affected)$" /sys/devices/system/cpu/vulnerabilities/spectre_v2`.

I think Python would also be suitable here, but due to these reasons and existing use of bash in the project (e.g. `devtool`) I went with this approach. I do not lean heavily either direction though so if you think it would be better suited in Python I could rewrite it as such.

Reply:
See https://github.com/firecracker-microvm/firecracker/pull/3036#discussion_r900152307